.github/workflows/python-build.yaml

```
@@ -15,8 +15,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.8]
-        poetry-version: [1.1.6]
+        python-version: [3.10]
+        poetry-version: [1.5.0]
```

```suggestion
        poetry-version: [1.4.1]
```
Can we use poetry 1.4.1. Our org is using 1.4.1 and we are using that for our other open source library as well.

---

sure, changed in f73ff94a733a50d1e9a6b0a87f5bbdf3705767fd